crates/floresta-chain/src/pruned_utreexo/mod.rs

```diff
@@ -362,9 +362,9 @@ impl<T: BlockchainInterface> BlockchainInterface for Arc<T> {
         proof: Proof,
         inputs: HashMap<OutPoint, TxOut>,
         del_hashes: Vec<sha256::Hash>,
-        acc: Stump,
+        validate_block: bool,
```

Comment:
The decision on whether we do validate or not should be decided by the [`verify_script`](https://github.com/Davidson-Souza/Floresta/blob/0b124684bb25031f01c07901c6c62377b6478361/crates/floresta-chain/src/pruned_utreexo/chain_state.rs#L720) method. We use this flag to implement `assumevalid`, where the scripts for transaction buried under enough blocks will not be validated, since ECC OPs are too expensive computationally.

Reply:
how i didnt know about this method ? KKKK